queries/zig/highlights.scm

```diff
@@ -152,71 +164,94 @@
   "while"
 ] @repeat
 
-(assignment_modifier) @attribute
+[
+  "or"
+  "and"
+  (BitwiseOp "orelse")
+] @keyword.operator
 
 [
-  ".{"
-  "("
-  ")"
-  "["
-  "]"
-  "{"
-  "}"
-] @punctuation.bracket
+  "struct"
+  "enum"
+  "union"
+  "error"
+  "packed"
+  "opaque"
+] @keyword
+
+"error" @exception
```

Comment:
What about `try` being exception?

Reply:
The hl for keyword i use this https://github.com/ziglang/zig.vim/blob/master/syntax/zig.vim as a reference. If this is make it better, i could make a change. Just wait me a bit

Reply:
I meant `try` could be `@keyword.exception` since it is a keyword related to exception/error handling.